indico/web/client/js/react/components/PrincipalSearchField.jsx

```diff
@@ -54,40 +54,67 @@ const searchGroups = async (data) => {
 export default class PrincipalSearchField extends React.Component {
     static propTypes = {
         multiple: PropTypes.bool,
-        defaultValue: PropTypes.object,
+        value: PropTypes.oneOfType([
+            PropTypes.object,
+            PropTypes.array
+        ]),
         onChange: PropTypes.func.isRequired,
         disabled: PropTypes.bool,
         withGroups: PropTypes.bool
     };
 
     static defaultProps = {
         multiple: false,
-        defaultValue: null,
+        value: null,
         disabled: false,
         withGroups: false
     };
 
     constructor(props) {
         super(props);
-        const {defaultValue} = this.props;
-        const userId = defaultValue && defaultValue.id;
+
+        const {multiple, onChange} = this.props;
+        let {value} = this.props;
+        let dropdownValue;
+
+        if (multiple) {
+            value = value || [];
+            dropdownValue = value && value.length !== 0 ? value.map((val) => val.id) : [];
```

Comment:
`value &&` isn't needed since you ensure it's an empty list in case of anything falsy one line above.